server/package.json

```diff
@@ -4,7 +4,8 @@
   "description": "Backend subproject for offline-markdown-previewer",
   "main": "server.js",
   "config": {
-    "port": "8080"
+    "http_port": "8080",
+    "ws_port": "8081"
```

Comment:
Actually, I was thinking about launching WebSocket server on top of our HTTP server, so they will use one port. http://stackoverflow.com/questions/20919947/serve-websocket-and-http-server-at-same-address-on-node-js something like this.

What do you think about this? Any caveats?